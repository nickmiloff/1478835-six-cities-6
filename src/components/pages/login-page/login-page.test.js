import React from 'react';
import {render} from '@testing-library/react';
import Test from '../../test/test';

import LoginPage from './login-page';

describe(`LoginPage component test`, () => {
  it(`'LoginPage component' should render correctly`, () => {
    const {container} = render(
        <Test>
          <LoginPage />
        </Test>
    );

    expect(container).toMatchSnapshot();
  });
});
