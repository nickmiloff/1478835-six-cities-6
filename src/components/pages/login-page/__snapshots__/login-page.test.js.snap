// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage component test 'LoginPage component' should render correctly 1`] = `
<div>
  <div
    class="page page--gray page--login"
  >
    <header
      class="header"
    >
      <div
        class="container"
      >
        <div
          class="header__wrapper"
        >
          <div
            class="header__left"
          >
            <a
              class="header__logo-link header__logo-link--active"
              href="/"
            >
              <img
                alt="6 cities logo"
                class="header__logo"
                height="41"
                src="img/logo.svg"
                width="81"
              />
            </a>
          </div>
          <nav
            class="header__nav"
          >
            <ul
              class="header__nav-list"
            >
              <li
                class="header__nav-item user"
              >
                <a
                  class="header__nav-link header__nav-link--profile"
                  href="/login"
                >
                  <div
                    class="header__avatar-wrapper user__avatar-wrapper"
                  />
                  <span
                    class="header__login"
                  >
                    Sign in
                  </span>
                </a>
              </li>
            </ul>
          </nav>
        </div>
      </div>
    </header>
    <main
      class="page__main page__main--login"
    >
      <div
        class="page__login-container container"
      >
        <section
          class="login"
        >
          <h1
            class="login__title"
          >
            Sign in
          </h1>
          <form
            action="#"
            class="login__form form"
            data-testid="form"
            method="post"
          >
            <div
              class="login__input-wrapper form__input-wrapper"
            >
              <label
                class="visually-hidden"
              >
                E-mail
              </label>
              <input
                autocomplete="on"
                class="login__input form__input"
                data-testid="email"
                name="email"
                placeholder="Email"
                required=""
                type="email"
                value=""
              />
            </div>
            <div
              class="login__input-wrapper form__input-wrapper"
            >
              <label
                class="visually-hidden"
              >
                Password
              </label>
              <input
                autocomplete="on"
                class="login__input form__input"
                data-testid="password"
                name="password"
                placeholder="Password"
                required=""
                type="password"
                value=""
              />
            </div>
            <button
              class="login__submit form__submit button"
              type="submit"
            >
              Sign in
            </button>
          </form>
        </section>
        <section
          class="locations locations--login locations--current"
        >
          <div
            class="locations__item"
          >
            <a
              class="locations__item-link"
              href="#"
            >
              <span>
                Amsterdam
              </span>
            </a>
          </div>
        </section>
      </div>
    </main>
  </div>
</div>
`;
